smurfs/src/actions/index.js

```diff
@@ -1,7 +1,54 @@
-/* 
-  Action Types Go Here!
-  Be sure to export each action type so you can pull it into your reducer
-*/
+import axios from 'axios';
+export const GET_SMURFS = 'GET_SMURFS';
+export const SMURFS_SUCCESS = 'SMURFS_SUCCESS';
+export const ADD_SMURF  = 'ADD_SMURF';
+export const SMURF_ADDED = 'SMURF_ADDED';
+export const SMURFS_FAILURE ='SMURFS_FAILURE';
+
+
```

Comment:
I like how you did the / at the end so that you can just do `${url}smurfs` later. I normally do not have the last / but then have to do something like  `${url}/smurfs`